annotation/src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
 vep.url=http://grch37.rest.ensembl.org/vep/human/hgvs/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1
+hotspots.url=http://cancerhotspots.org/api/hotspots/single/
+vep.overrides_uris=uniprot=isoform_overrides_uniprot.txt,mskcc=isoform_overrides_at_mskcc.txt
```

Comment:
@zheins sounds good to me. I shouldn't have included this property in this PR actually. It is part of the isoform support, not related to cancer hotspots. I will change it in the next isoform support PR.